go.mod

```diff
@@ -63,7 +63,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.2-0.20220504104629-106ec21d14df
 	github.com/tiancaiamao/appdash v0.0.0-20181126055449-889f96f722a2
-	github.com/tikv/client-go/v2 v2.0.1-0.20220531092439-efebaeb9fe53
```

Comment:
Don't forget to update bazel's dependencies.

Reply:
Release 6.1 doesn't seem to have bazel build files.

Reply:
Sorry, I don't care about it.